README.md

```diff
@@ -12,6 +12,8 @@
 
 [![FOSSA Status](https://app.fossa.io/api/projects/git%2Bgithub.com%2Ftechqueria%2Fwebsite.svg?type=small)](https://app.fossa.io/projects/git%2Bgithub.com%2Ftechqueria%2Fwebsite?ref=badge_small)
 
+[![first-timers-only](https://img.shields.io/badge/first--timers--only-friendly-blue.svg?style=flat-square)](https://www.firsttimersonly.com/)
```

Comment:
@sonali9696 can we remove `?style=flat-square` so it matches format of other badges and also move it to be above FOSSA status?

Reply:
Made the required changes